make.js

```diff
@@ -988,25 +827,18 @@ target.mozcentral = function() {
            ROOT_DIR + MOZCENTRAL_EXTENSION_DIR + '/chrome.manifest');
   cd(ROOT_DIR);
 
-  var TMP_VIEWER = MOZCENTRAL_CONTENT_DIR + '/web/viewer.js.tmp';
-  cd('web/');
-  var viewerBundleFiles = ['app.js', 'firefoxcom.js'];
-  bundle('viewer.js', ROOT_DIR + TMP_VIEWER, viewerBundleFiles,
-    'pdfjs-dist/web/viewer', defines, true);
-  cd(ROOT_DIR);
-
   var setup = {
     defines: defines,
     copy: [
+      [BUILD_TARGETS, MOZCENTRAL_CONTENT_DIR + BUILD_DIR],
       [COMMON_WEB_FILES, MOZCENTRAL_CONTENT_DIR + '/web'],
       ['external/bcmaps/*', MOZCENTRAL_CONTENT_DIR + '/web/cmaps'],
       ['extensions/firefox/tools/l10n.js', MOZCENTRAL_CONTENT_DIR + '/web']
     ],
     preprocess: [
-      [TMP_VIEWER, MOZCENTRAL_CONTENT_DIR + '/web/viewer.js'],
       [COMMON_WEB_FILES_PREPROCESS, MOZCENTRAL_CONTENT_DIR + '/web'],
+      [BUILD_DIR + 'viewer.js', MOZCENTRAL_CONTENT_DIR + '/web'],
```

Comment:
Why can't we move this to the `copy` section above, just like the other targets?